src/uct/sm/mm/mm_posix.c

```diff
@@ -17,12 +17,16 @@
 #define UCT_MM_POSIX_MMAP_PROT      (PROT_READ | PROT_WRITE)
 #define UCT_MM_POSIX_HUGETLB        UCS_BIT(0)
 #define UCT_MM_POSIX_SHM_OPEN       UCS_BIT(1)
-#define UCT_MM_POSIX_CTRL_BITS      2
+#define UCT_MM_POSIX_PROC_LINK      UCS_BIT(2)
+#define UCT_MM_POSIX_CTRL_BITS      3
+#define UCT_MM_POSIX_FD_BITS        29
```

Comment:
maybe we should add a ucs_assert in the alloc function to make sure that shm_fd and shm_fd<<3 are equal? or we are sure that it's always the case? 
what do you think?